### DISCUSSION TITLE
socketify as a backend for fastapi?

### DISCUSSION THREAD

**Discussion by bitnom** [fastapi/fastapi#10827] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
NA
```

### Description

Is there a possibility of integrating socketify as a backend for fastapi?

I'm not a contributor to either of these projects, nor have I been using fastapi for that long (I recently switched to it from another), so please forgive my ignorance if this is a dumb question.

### Operating System

Linux

### Operating System Details

NA

### FastAPI Version

NA

### Pydantic Version

NA

### Python Version

NA

**Comment by Kludex** [comment, marked as answer]
You can use socketify as an ASGI server: https://docs.socketify.dev/cli.html
